lib/editor/components/map/LocationsLayer.js

```diff
@@ -115,36 +119,32 @@ export default class LocationsLayer extends PureComponent<Props, State> {
 
           // Polygons have 2D latlng object structure
           // $FlowFixMe flow doesn't understand that arrays are objects. Can this typecheck be done cleaner?
-          const layerCoords = isPolygon ? coordSet[0].map(latlng => [latlng.lat, latlng.lng]) : coordSet.map(latlng => [latlng.lat, latlng.lng])
+          const layerCoords = isPolygon ? coordSet[0].map(convertLatLngToArray) : coordSet.map(convertLatLngToArray)
```

Comment:
I think it will be useful to extract the construct to a util function because it appears multiple times.
```js
isPolygon ? coordSet[0] : coordSet
```

Reply:
@philip-cline do you know what Binh means by this

Reply:
I think this makes sense, creating a util function that takes the coordSet and applies the mapping and returns the coords for if its a polygon or polyline